third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -24,19 +24,21 @@ def arcs_kt_binary(name, srcs = [], deps = [], visibility = None):
     """Performs final compilation of wasm and bundling if necessary."""
     libname = name + "_lib"
```

Comment:
Minor nit: move this def inside the if-statement below

Reply:
Done

Reply:
Done